contracts/examples/DXswapRelayer.sol

```diff
@@ -188,29 +190,21 @@ contract DXswapRelayer {
     function executeOrder(uint256 orderIndex) external {
         Order storage order = orders[orderIndex];
         require(orderIndex <= orderCount && orderIndex != 0, 'DXswapRelayer: INVALID_ORDER');
-        require(order.executed == false, 'DXswapRelayer: ORDER_EXECUTED');
+        require(!order.executed, 'DXswapRelayer: ORDER_EXECUTED');
         require(oracleCreator.isOracleFinalized(order.oracleId) , 'DXswapRelayer: OBSERVATION_RUNNING');
         require(block.timestamp <= order.deadline, 'DXswapRelayer: DEADLINE_REACHED');
 
         address tokenA = order.tokenA;
         address tokenB = order.tokenB;
         uint256 amountB;
-        if(tokenA == address(0)){
-            amountB = oracleCreator.consult(
-              order.oracleId,
-              IDXswapRouter(dxSwapRouter).WETH(),
-              order.amountA
-            );
-        } else {
-            amountB = oracleCreator.consult(
-              order.oracleId,
-              tokenA,
-              order.amountA
-            );
-        }
+        amountB = oracleCreator.consult(
+          order.oracleId,
+          tokenA == address(0) ? IDXswapRouter(dxSwapRouter).WETH() : tokenA,
```

Comment:
Is it necessary to use WETH when address zero? why dont you just use WETH?, you have the address available in the contract.